data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/ConvertEntryTypeProcessorConfig.java

```diff
@@ -80,6 +80,11 @@ public class ConvertEntryTypeProcessorConfig implements ConverterArguments {
     })
     private String convertWhen;
 
+    @JsonProperty("iterate_on")
+    @JsonPropertyDescription(
+            "Specifies the key of a list of objects to iterate over and convert types.")
```

Comment:
Should it be list of `map objects` or something like that?

Reply:
objects implies Map but I can add `map objects`